Simplified/NYPLBarcodeScanningViewController.m

```diff
@@ -85,18 +85,13 @@ - (void)didSelectCancel
   [self dismissViewControllerAnimated:YES completion:nil];
 }
 
-// TODO: remove after fixing SIMPLY-2590
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-implementations"
-- (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrientation {
-  return toInterfaceOrientation == UIInterfaceOrientationPortrait;
+- (UIInterfaceOrientationMask)supportedInterfaceOrientations {
+    return UIInterfaceOrientationMaskPortrait;
 }
 
-- (void)didRotateFromInterfaceOrientation:(UIInterfaceOrientation)fromInterfaceOrientation {
-	[super didRotateFromInterfaceOrientation:fromInterfaceOrientation];
-	[self applyOrientation];
```

Comment:
just checking, we lost the call to `applyOrientation` here. Did you verify this is still being called?

Reply:
Yeah it's being called in `viewWillTransitionToSize:withTransitionCoordinator:`